pkgs/development/python-modules/pdfrw/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, mupdf, swig }:
+buildPythonPackage rec {
+  pname = "pdfrw";
+  version = "0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1x1yp63lg3jxpg9igw8lh5rc51q353ifsa1bailb4qb51r54kh0d";
+  };
+
+  # no python3 support yet
+  doCheck = false;
```

if the comment is true, then this should work
```suggestion
  doCheck = isPy27;
```

---

turns out that there are even more errors with python2 so I just disabled for both as originally ( I had not tested python2 at the time)

---

there should be an attempt to run all the pure tests that you can. There will be some that you can't do in a sandbox environment, but we want to guarentee that the package wont break when your dependencies get bumped.

---

it requires additionnal pdfs https://github.com/pmaupin/pdfrw#6testing . I think it's fine to disable them. The decision can be revisited when and if the package breaks.